diff -up glew-glew-2.3.1/auto/src/glew_head.c.4~ glew-glew-2.3.1/auto/src/glew_head.c
--- glew-glew-2.3.1/auto/src/glew_head.c.4~	2026-01-24 00:55:35.000000000 +0100
+++ glew-glew-2.3.1/auto/src/glew_head.c	2026-04-01 16:35:02.741973319 +0200
@@ -16,6 +16,7 @@
 #    undef GLEW_APIENTRY_DEFINED
 #  endif
 #elif defined(GLEW_EGL)
+#  include <EGL/egl.h>
 #  include <GL/eglew.h>
 #elif defined(_WIN32)
 /*
