Index: glew-2.0.0/Makefile
===================================================================
--- glew-2.0.0/Makefile
+++ glew-2.0.0/Makefile	2017-07-17 15:09:46.816288394 +0200
@@ -45,7 +45,7 @@
 BINDIR    ?= $(GLEW_DEST)/bin
 LIBDIR    ?= $(GLEW_DEST)/lib
 INCDIR    ?= $(GLEW_DEST)/include/GL
-PKGDIR    ?= $(GLEW_DEST)/lib/pkgconfig
+PKGDIR    ?= $(LIBDIR)/pkgconfig
 
 ifneq ($(GLEW_NO_GLU), -DGLEW_NO_GLU)
 LIBGLU = glu
